### ISSUE TITLE
Tencent.TencentMeeting reports it installed but actually nothing

### ISSUE THREAD

**Issue by lychichem** [microsoft/winget-pkgs#14609]
Create issues related to the winget.exe client [here](https://github.com/microsoft/winget-cli/issues/new?template=Bug_Report.md)

# Brief description of your issue
Tencent.TencentMeeting reported it installed but actually it installed nothing.
# Steps to reproduce
Install Tencent.TencentMeeting with `winget install Tencent.TencentMeeting`

# Expected behavior
The program should be installed properly

# Actual behavior
The app actually installed nothing - no desktop icon, no start menu folder and even no app folder.

# Environment

```none
[winget --info]
Windows Package Manager version: v1.0.11451
Windows: Windows.Desktop version: 10.0.19043.1023 x64 pro zh-CN
Package: Microsoft.DesktopAppInstaller version: 1.11.11451.0

Any other software?
```
Besides, When running the latest version (2.12.3.408) installer directly with /S switch in cmd, it also not properly installed. It will not create icon anywhere, and will not register uninstaller string in ARP list. So, I suggest remove it unless someone find the correct switch to install it.

**Comment by GeorgeCY2** [comment]
sounds like it is an installer end issue. That can mean that essentially Tencent Meeting isnt properly supporting silent install despite the package manager trying to do so.

**Comment by lychichem** [comment]
> sounds like it is an installer end issue. That can mean that essentially Tencent Meeting isnt properly supporting silent install despite the package manager trying to do so. Try debugging things like taking a stable or correct installer file, try installing for yourself and see what could be causing it? in the end again what i said and maybe try debugging things to see if anything can make it work.

It may related to wrong switch or poor designed script used to build the installer. Anyway, switch /S cannot properly install the latest version of tencentmeeting, although the program files are released to current user folder.

**Comment by GeorgeCY2** [comment]
> It may related to wrong switch or poor designed script used to build the installer. Anyway, switch /S cannot properly install the latest version of tencentmeeting, although the program files are released to current user folder. Also, I cannot get the document about the install switch or build script of tencent meeting, so I cannot figure out this issue.
understandable. Again it sounds like the silent install is not installing correctly and unfortuantely that's gotta be up to tencent to patch it in.
Or you can submit a newer version and try it because it sounds like tencent has a newer version looking at the download file here:
https://updatecdn.meeting.qq.com/da697ac497db008ba085899ff5615282/VooVMeeting_1410000197_2.10.2.510.publish.exe
the package manager has latest as 2.6... maybe you can make a manifest and see if they patched it this way or installs better

**Comment by lychichem** [comment]
> > It may related to wrong switch or poor designed script used to build the installer. Anyway, switch /S cannot properly install the latest version of tencentmeeting, although the program files are released to current user folder. Also, I cannot get the document about the install switch or build script of tencent meeting, so I cannot figure out this issue.
> 
> understandable. Again it sounds like the silent install is not installing correctly and unfortuantely that's gotta be up to tencent to patch it in.
> Or you can submit a newer version and try it because it sounds like tencent has a newer version looking at the download file here:
> https://updatecdn.meeting.qq.com/da697ac497db008ba085899ff5615282/VooVMeeting_1410000197_2.10.2.510.publish.exe
> the package manager has latest as 2.6... maybe you can make a manifest and see if they patched it this way or installs better

I've tried 2.12.3.408 (As a Chinese company, Chinese version of tencent meeting has higher version number). And the situation I discribed is just what happens to THIS installer. It is better than package listed in winget (At least it can "install" the main program, although I have to manually run the uninstaller to uninstall it), but a good installer should do more.

**Comment by denelon** [comment]
I believe this is an issue due to the installer. I'm going to mark this as external until or unless we can confirm it is a problem with the Windows Package Manager client. Feel free to refer the publisher back to this issue so they can test and validate the behavior.

**Comment by lychichem** [comment]
Close because it's really a bug related to poor-designed installer. The installer requires admin priviledge but it don't even try to acquire it and give no report. @denelon Can I create a .bat file, which use curl to donloads the installer from the same link but add a part to get priviledge before run the installer, and distribute this batch file in winget?

**Comment by denelon** [comment]
We've been making changes to the schema. We've added a "app requires admin to install" capability in the schema. As soon as it's implemented in the client, we can address this issue in the manifest.

**Comment by lychichem** [comment]
> We've been making changes to the schema. We've added a "app requires admin to install" capability in the schema. As soon as it's implemented in the client, we can address this issue in the manifest.

I didn't see it in the doc, if it was added, could you please update the docs to let manifest writers know how to use it?

**Comment by denelon** [comment]
@lychichem modifying the schema is a fairly long process. We will have support for validating the new schema ahead of any client support for the behavior. I'll be updating the docs once all of the changes are actually merged into a release. 

We will also have delays on the validation side before accepting any of the new schema elements into the repository. The new "Agreements" node will be restricted to "verified publishers" which is waiting on some additional business process work. Some of the new schema elements are also forward looking so the client implementation will also be delayed. If you look at the current documentation, you will see several YAML keys aren't associated with any behavior in the client (like channels).
